kodi_addon_checker/handle_files.py

```diff
@@ -46,10 +46,14 @@ def create_file_index(path: str):
     """
     file_index = []
     for root, folders, files in os.walk(path, topdown=True):
+        if ".directory" in files:
```

Comment:
Why only skip directories named `.directory`?

Reply:
You mean to say we should ignore all the hidden files in the root directory?

Reply:
What is special about a folder named `.directory` that it should be ignored, but e.g. not a folder named `.abc`?
The current code does ignore `.directory` folders everywhere, not only in the root directory.
The current PR title is "Ignore hidden files and folders", which means ignore all files and folder for me.

Reply:
@Razzeee @mzfr I still don't get the reason to exclude a file named `.directory`. What's this special file that it shouldn't be listed in the file index?

Reply:
This file is basically used by file managers to keep track of the direcotries and also used for the preview of the direcotries.

Why I decided to ignore them? Since these files might be present in the folders(root) of addon but they don't consist any data that can be used by an addon so I simply decided to ignore them.

Reply:
So it's like `.DS_Store` and `Thumbs.db` on other OSes?

Still the checker shouldn't ignore the file, as  the checker now doesn't warn about those files submitted in an add-on submission.